<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceWidget</class>
 <widget class="QWidget" name="DeviceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>